doc/working-with-bigints.md

```diff
@@ -20,7 +20,7 @@ code:
 -   Right now unfortunately mathjs does not work on bigints but soon we will
```

Comment:
maybe this page should be linked from math.md

Reply:
also, right above here we see the phrase "consist digits" (there should be "of")